functional-tests/fn_bridge_deposit_reclaim_drt_seen.py

```diff
@@ -147,7 +143,8 @@ def main(self, ctx: flexitest.RunContext):
             btc_password,
         )
         self.debug(f"User BTC balance (after takeback): {refund_btc_balance}")
-        expected_balance = 5 * DEPOSIT_AMOUNT - TAKE_BACK_FEE
+        deposit_amount = ctx.env.rollup_cfg().deposit_amount
```

Comment:
Maybe we should normalize like below?

```suggestion
        cfg: RollupConfig = ctx.env.rollup_cfg()
        deposit_amount = cfg.deposit_amount
```

Reply:
Agree, done